scrapy/http/cookies.py

```diff
@@ -1,10 +1,14 @@
+import re
 import time
-from http.cookiejar import CookieJar as _CookieJar, DefaultCookiePolicy, IPV4_RE
+from http.cookiejar import CookieJar as _CookieJar, DefaultCookiePolicy
 
 from scrapy.utils.httpobj import urlparse_cached
 from scrapy.utils.python import to_unicode
 
 
+IPV4_RE = re.compile(r"\.\d+$", re.ASCII)
```

Comment:
Taken from https://github.com/python/cpython/blob/3.9/Lib/http/cookiejar.py#L527

`IPV4_RE` is neither in [`__all__`](https://github.com/python/cpython/blob/3.9/Lib/http/cookiejar.py#L28-L29) nor in the [docs](https://docs.python.org/3/library/http.cookiejar.html).

Reply:
Would you mind adding something simiilar as a code comment?